server/migrations/functions.py

```diff
@@ -0,0 +1,29 @@
+# PostgreSQL functions that are applied, that needs to be both migrated and recreated
+# in our tests.
+
+SEQUENCE_CREATE_CUSTOMER_SHORT_ID = "CREATE SEQUENCE customer_short_id_seq START 1;"
+
+FUNCTION_GENERATE_CUSTOMER_SHORT_ID = """
+CREATE OR REPLACE FUNCTION generate_customer_short_id()
+RETURNS bigint AS $$
+DECLARE
```

Comment:
This is based on the Instagram ID generation algorithm. We get fairly small, ordered but not leaking user volume, IDs that we can base26-encode to get an alphabetical identifier we can use for invoice numbers.

Reply:
Exactly, we'd set it to 0 or 1 for now, and then a component of the id would just be fixed. It would allow us to introduce it at some point without migrating the IDs or having two realities.